src/geodesic.rs

```diff
@@ -830,7 +827,7 @@ impl Geodesic {
                 let A4 = geomath::sq(self.a) * calp0 * salp0 * self._e2;
                 geomath::norm(&mut ssig1, &mut csig1);
                 geomath::norm(&mut ssig2, &mut csig2);
-                let mut C4a: [f64; CARR_SIZE] = [0.0; CARR_SIZE];
+                let mut C4a: [f64; GEODESIC_ORDER as usize] = [0.0; GEODESIC_ORDER as usize];
                 self._C4f(eps, &mut C4a);
```

Comment:
yet another spot where C++ just uses the same Ca. 🤷 

I again agree your change is an improvement, since we're at least be consistent with *some* implementation.